pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -123,6 +127,38 @@ func (pl *VolumeBinding) EventsToRegister() []framework.ClusterEventWithHint {
 	return events
 }
 
+func (pl *VolumeBinding) isSchedulableAfterCSINodeChange(logger klog.Logger, pod *v1.Pod, oldObj, newObj interface{}) (framework.QueueingHint, error) {
```

Comment:
Checking whether a pod has a migratable persistent volume may be a time-consuming operation for hint function. so I agree with the current implementation.

Reply:
There's a to-do item. it is that the kube-schduler should unregister the csinode events once all the volume plugins has completed their csi migration. 

cc @sanposhiho

Reply:
https://github.com/kubernetes/kubernetes/blob/5d1a97ce092b1c1cbb3dc40b5a7e4dffe4b8f1db/pkg/volume/csi/csi_plugin.go#L314

Reply:
@carlory 
I add todo comment!
[207eb08](https://github.com/kubernetes/kubernetes/pull/125097/commits/207eb08a8d82affb8832a71afc05410bf03ec15b)